backend/redis/health_redis.go

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2019 by PACE Telematics GmbH. All rights reserved.
+// Created at 2019/10/18 by Charlotte Pröller
+
+package redis
+
+import (
+	"time"
+
+	"github.com/go-redis/redis"
+	"github.com/pace/bricks/maintenance/health/servicehealthcheck"
+)
+
+// HealthCheck checks the state of a redis connection. It must not be changed
+// after it was registered as a health check.
+type HealthCheck struct {
+	state      servicehealthcheck.ConnectionState
+	Client     *redis.Client
+	CheckWrite bool
```

Comment:
Why is this calles `CheckWrite` when the default healthcheck in `HealthCheck()` is always writing?